azure_sdk_core/src/headers.rs

```diff
@@ -35,3 +35,4 @@ pub const DELETE_TYPE_PERMANENT: &str = "x-ms-delete-type-permanent";
 pub const DELETE_SNAPSHOTS: &str = "x-ms-delete-snapshots";
 pub const SKU_NAME: &str = "x-ms-sku-name";
 pub const ACCOUNT_KIND: &str = "x-ms-account-kind";
+pub const APPEND_POSITION: &str = "x-ms-blob-appendpos";
```

Comment:
I cannot find this header in the docs. The only thing I can find is `x-ms-blob-condition-appendpos` which, from what I gather, is precondition test. Do you mean to specify that?

Reply:
Whoops, just a typo. Thanks for the catch!